src/SpectatorView.Unity/Assets/SpatialAlignment.ASA/Scripts/SpatialAlignment/SpatialAnchorsCoordinateService.cs

```diff
@@ -324,6 +324,11 @@ private async Task<ISpatialCoordinate> TryCreateCoordinateImplAsync(Vector3 worl
                     await session.CreateAnchorAsync(cloudSpatialAnchor);
                     return new SpatialAnchorsCoordinate(cloudSpatialAnchor, spawnedAnchorObject);
                 }
+                catch(TaskCanceledException)
```

Comment:
I would recommend making this a OperationCancelledException, which is the base of TaskCancelledException